lib/build.api.js

```diff
@@ -145,6 +145,17 @@ class Build {
           }
         });
 
+        // Add config links.
+        Object.keys(project.config.paths.configDirectories).forEach((key) => {
```

Comment:
@thebruce Nice work. Just a couple of things to think about. Will `project.config.paths.configDirectories` always exist? If it doesn't this will throw an error and everything else will fail. We may want to check each of `project.config.paths.*` keys to make sure they exist.

It also looks like we are basically doing the same thing three different times. Could we wrap this up into a function and put it outside this main chunk of code? If we ever get around to adding tests, we'll already be better off. 

-Matt 🍰